pkg/integrations/register.go

```diff
@@ -98,20 +98,23 @@ func MarshalYAML(v interface{}) (interface{}, error) {
 	for i, n := 0, inType.NumField(); i < n; i++ {
 		if inType.Field(i).Type == configsType {
 			configs = inVal.Field(i).Interface().(Configs)
+			if configs == nil {
+				configs = Configs{}
+			}
```

Comment:
The slice defaults to nil, so we have to do this here to make sure the function returns without erroring.